src/ScriptCs.Core/App_Packages/LibLog.3.1/LibLog.cs

```diff
@@ -0,0 +1,1941 @@
+//===============================================================================
```

Comment:
I made a conscious decision to ship as a single file. Don't want to crap stuff all over peoples' csprojs. Also helps reduce chances of accidental editing / refactoring. If you _do_ edit it, an update to a newer version won't work (smoothly). In my experience of using source code packages, I've found life is easier this way.

While it's also developed as a single file but even if I split that out I'd still ship as a single file.